### ISSUE TITLE
Unable to set a database notification template

### ISSUE THREAD

**Issue by robinsm** [solspace/craft3-freeform#1525]
### What happened?

Unable to set a database notification template when editing forms.

This problem can be resolved when manually adding a getter for the `uid` property to `Solspace\\Freeform\\Library\\DataObjects\\NotificationTemplate`

<img width="1563" alt="Contact_Us" src="https://github.com/user-attachments/assets/cbeb8eca-4cf0-42f5-b6ea-ad1ed54acd21">

### Errors and Stack Trace (if available)

```shell
"Can't get a way to read the property \"uid\" in class \"Solspace\\Freeform\\Library\\DataObjects\\NotificationTemplate\"."
exception: "Symfony\\Component\\PropertyAccess\\Exception\\NoSuchPropertyException"
file: "/var/www/html/vendor/symfony/property-access/PropertyAccessor.php"
line: 449
```

### How can we reproduce this?

1. Use only database notification templates
2. Create at least one database template
3. Edit existing form
4. Move to notifications tab
5. Attempt to select a notification template.

### Freeform Edition

Pro

### Freeform Version

5.6.1

### Craft Version

4.11.4

### When did this issue start?

After upgrading from older Freeform version

### Previous Freeform Version

4.x

**Comment by gustavs-gutmanis** [comment]
Thanks for reporting this @robinsm!

We'll release a fix for this shortly.

**Comment by zenbug** [comment]
I'm encountering this issue with flat-file notification templates as well, not just database templates.

**Comment by kjmartens** [comment]
This should now be resolved in Freeform 5.6.2+ 🙂
